test/standalone/test-preloaded-agent.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var semver = require('semver');
+var execSync = require('child_process').execSync;
+var assert = require('assert');
+
+if (semver.satisfies(process.versions.node, '>= 3.x')) {
```

Comment:
I am actually going to change this to be >= 2.2.0, because that's when I actually added `internal/preload`.
https://github.com/nodejs/node/blob/master/CHANGELOG.md#2015-05-31-version-220-rvagg